projects/components/src/dropdown/dropdown.component.html

```diff
@@ -36,6 +36,7 @@
                 <ng-container *ngIf="!action.children">
                     <button
                         class="btn btn-link"
+                        [className]="action.class"
```

Comment:
We're already using `ngClass`, why use something else?

Reply:
ngClass lets me add a string value conditionally. But here, I want to ass a property value as a class name. that is why I did this.

Reply:
It seems we should replace class with 

```
[className]="[action.class, 'btn', 'btn-link']"
```